src/main/java/core/basesyntax/AvailableFigure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+enum AvailableFigure {
```

Comment:
Enums are typically constants and should be named using uppercase letters. For example, 'RECTANGLE' should be 'RECTANGLE', 'CIRCLE' should be 'CIRCLE', and so on.

Reply:
Explain to me how RECTANGLE is different from RECTANGLE